src/index.css

```diff
@@ -14,7 +14,7 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-  height: 100vh;
+  height: 100%;
   background-color: #f3f4f8;
 }
```

Comment:
- [ ] According to the project requirement you should use the exact design as provided. please make your `completed section` in the middle. and also use the same color and font for the `current chapter` section as well. See the image below for reference.
### Your design
![his](https://user-images.githubusercontent.com/81727988/216776805-2aabc48e-5e6d-4f6c-887c-2d473cfd94d4.png)

### Original design
![orignial](https://user-images.githubusercontent.com/81727988/216776818-846d5cf7-1fed-4cdb-a491-dc283dbccb41.png)